examples/mia/tabular_mia/main.ipynb

```diff
@@ -0,0 +1,533 @@
+      "Preparing MIA data trajectory_train_data.pkl: 100%|██████████| 85/85 [00:05<00:00, 15.54it/s]\n",
+      "2024-09-10 14:02:44,750 INFO     Preparing MIA trajectory_test_data.pkl: 27132 points\n",
+      "Preparing MIA data trajectory_test_data.pkl: 100%|██████████| 424/424 [00:28<00:00, 15.03it/s]\n",
+      "2024-09-10 14:03:13,265 INFO     Running attack: loss_traj\n",
+      "2024-09-10 14:03:13,266 INFO     Training the MIA classifier\n",
+      "100%|██████████| 100/100 [00:18<00:00,  5.33it/s]\n",
+      "2024-09-10 14:03:32,027 INFO     Running the MIA attack\n",
+      "100%|██████████| 424/424 [00:00<00:00, 1193.89it/s]\n",
+      "2024-09-10 14:03:32,753 INFO     Finished attack: loss_traj\n",
+      "2024-09-10 14:03:32,754 INFO     Preparing results for attack: rmia\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No existing file named './leakpro_output/results/rmia/results.txt'. A new file will be created.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-09-10 14:03:36,075 INFO     Preparing results for attack: qmia\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No existing file named './leakpro_output/results/qmia/results.txt'. A new file will be created.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-09-10 14:03:39,396 INFO     Preparing results for attack: population\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No existing file named './leakpro_output/results/population/results.txt'. A new file will be created.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-09-10 14:03:42,678 INFO     Preparing results for attack: lira\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No existing file named './leakpro_output/results/lira/results.txt'. A new file will be created.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-09-10 14:03:46,089 INFO     Preparing results for attack: loss_traj\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No existing file named './leakpro_output/results/loss_traj/results.txt'. A new file will be created.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-09-10 14:03:49,831 INFO     Auditing completed\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<Figure size 640x480 with 0 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from adult_handler import AdultInputHandler\n",
```

Comment:
I get the point of having these imports here, but I still think it is better to move them up and instead write more in the markdown above this cell.